Description: Fix errors in the man pages
 * opagetvf_env.manPage - illegal character and line starting with period
 * opacmdall.manPage - line starting with period
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Forwarded: https://github.com/intel/opa-ff/pull/17
Last-Update: <2018-10-03>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/IbaTools/man/opagetvf_env.manPage
+++ b/IbaTools/man/opagetvf_env.manPage
@@ -82,7 +82,7 @@
 
 .SH Function Example
 .NL
-. /usr/sbin/opagetvf\(ulenv
+\[char46] /usr/sbin/opagetvf_env
 .br
 
 .br
--- a/IbaTools/man/opacmdall.manPage
+++ b/IbaTools/man/opacmdall.manPage
@@ -176,9 +176,7 @@
 
 .PP
 All commands performed with opacmdall must be non-interactive in nature. opacmdall waits for the command to complete before proceeding. For example, when running host commands such as rm, the -i option (interactively prompt before removal) should not be used. (Note that this option is sometimes part of a standard bash alias list.) Similarly, when running chassis commands such as fwUpdateChassis, the -reboot option should not be used because this option causes an immediate reboot and therefore the command never returns. Also, the chassis command reboot should not be executed using opacmdall. Instead, use the opachassisadmin reboot command to reboot one or more chassis. For further information about individual chassis CLI commands, consult the 
-\fIIntel(R) Omni-Path Fabric Switches Command Line Interface Reference Guide\fR
-
-. For further information about Linux* operating system commands, consult the man pages.
+\fIIntel(R) Omni-Path Fabric Switches Command Line Interface Reference Guide\fR. For further information about Linux* operating system commands, consult the man pages.
 .PP
 When performing opacmdall against hosts, internally SSH is used. The command opacmdall requires that password-less SSH be set up between the host running the Intel(R) Omni-Path Fabric Suite FastFabric Toolset and the hosts opacmdall is operating against. The opasetupssh FastFabric tool can aid in setting up password-less SSH.
 .PP
